stage2.py

```diff
@@ -4,7 +4,6 @@
 import re
 import shutil
 import subprocess
-import sys
 import tempfile
 import types
```

Comment:
You can remove this import since you're getting rid of uses of types.StringType.

Reply:
By "this" I meant "types"